plugins/catalog/src/components/DependencyOfComponentsCard/DependencyOfComponentsCard.test.tsx

```diff
@@ -21,28 +21,20 @@ import {
   EntityProvider,
   entityRouteRef,
 } from '@backstage/plugin-catalog-react';
-import { renderInTestApp } from '@backstage/test-utils';
+import { renderInTestApp, TestApiProvider } from '@backstage/test-utils';
 import { waitFor } from '@testing-library/react';
 import React from 'react';
 import { DependencyOfComponentsCard } from './DependencyOfComponentsCard';
-import { ApiProvider, ApiRegistry } from '@backstage/core-app-api';
 
 describe('<DependencyOfComponentsCard />', () => {
-  const catalogApi: jest.Mocked<CatalogApi> = {
-    getLocationById: jest.fn(),
-    getEntityByName: jest.fn(),
-    getEntities: jest.fn(),
-    addLocation: jest.fn(),
-    getLocationByEntity: jest.fn(),
-    removeEntityByUid: jest.fn(),
-  } as any;
+  const getEntities: jest.MockedFunction<CatalogApi['getEntities']> = jest.fn();
```

Comment:
This is something you can do now, which I think is quite a lot nicer tbh, with just providing `{ getEntities }` below

Reply:
This is huge! ☝️